src/voodoo-prep/main.ml

```diff
@@ -32,7 +32,9 @@ let process_package : Fpath.t -> Package.t -> Fpath.t list -> unit =
     let no_ext = Fpath.rem_ext filename in
     let has_hyphen = String.contains (Fpath.to_string filename) '-' in
     let is_module =
-      process_ocaml_artefacts && List.mem ext [ ".cmt"; ".cmti"; ".cmi" ] && not has_hyphen
+      process_ocaml_artefacts
```

Comment:
This change will unfortunately cause a full rebuild of the preparation step. It would be nice to avoid it.

Reply:
Thanks, I've reverted the changes on voodoo-do and voodoo-prep in d2b485e. We can revert this commit when we've confirmed that the pipeline runs fine, so let's remember to not squash the PR.